### ISSUE TITLE
Switch to Symfony Yaml

### ISSUE THREAD

**Issue by distantnative** [getkirby/kirby#2452]
Because of issues like https://github.com/getkirby/kirby/issues/1308
(see also there for the discussion).

- [x] Add additional unit tests for YAML
- [x] Switch out libraries
- [x] Test Symfony Yaml against all unit tests
- [x] Make sure Symfony Yaml resolves #1308

**Comment by tobiastom** [comment]
I just found this issue, and would like to mention https://github.com/getkirby/kirby/pull/2632 briefly. 

We also replaced the YAML handler with a different one (https://pecl.php.net/package/yaml). During this process I recognized that things **seem** to work, but internally the wrong parser was still used. After I changed this (the result is my mentioned pull request), I detected more incompatibility problems (especially in structured fields).

With this request we only needed to change a very small amount of content files to be supported by the new parser.
